cmd/soroban-rpc/internal/ingest/service.go

```diff
@@ -81,7 +82,11 @@ func NewService(cfg Config) *Service {
 		ledgerStatsMetric:       ledgerStatsMetric,
 	}
 	service.wg.Add(1)
-	go func() {
+	util.MonitoredRoutine(util.MonitoredRoutineConfiguration{
+		Log:                cfg.Logger,
+		LogPanicsToStdErr:  true,
```

Comment:
It seems like logging to stdout/stderr/wherever is a logger concern. E.g. if you wanted logging to stdout you'd do:
```go
util.MonitoredRoutineConfiguration{ Log: stderrLogger  }
```